definitions/features/hammer.rb

```diff
@@ -99,24 +101,24 @@ def ask_password_and_check(custom_config)
 
   def config_error
     raise ForemanMaintain::HammerConfigurationError, 'Hammer configuration failed: '\
-                  "Is the admin password correct? (it was stored in #{custom_config_file})" \
+                  'Is the admin username and password correct? ' \
```

Comment:
Can we reword this message something like 
` "Is the admin credential from the file #{custom_config_file} correct?\n" \
   'Is the server down?'`